app/views/browse/new_show.html.erb

```diff
@@ -6,7 +6,8 @@
   <%= render "govuk_publishing_components/components/meta_tags", {
     content_item: {
       # TODO align this naming "level 1" vs "second_level" mismatch
```

Comment:
Nit: I'm pretty sure we can remove this comment, I think the fix here is that this refers to.

Reply:
ah yes good spot